django_rq/tests/utils.py

```diff
@@ -5,15 +5,18 @@ def get_queue_index(name='default'):
     """
     Returns the position of Queue for the named queue in QUEUES_LIST
     """
-    queue_index = None
     connection = get_connection(name)
     connection_kwargs = connection.connection_pool.connection_kwargs
+
     for i in range(0, 100):
         try:
             q = get_queue_by_index(i)
         except AttributeError:
             continue
-        if q.name == name and q.connection.connection_pool.connection_kwargs == connection_kwargs:
-            queue_index = i
-            break
-    return queue_index
+        if q.name == name:
```

Comment:
I updated this to only check the name, since this method was skipping the queue (and breaking on an `IndexError` if the connection wasn't the "same")